packages/react/src/textarea/Textarea.css.ts

```diff
@@ -0,0 +1,53 @@
+import { recipe } from "../vanilla-extract";
+
+export const textAreaBoxRecipe = recipe({
+  base: {
+    background: "transparent",
+    flexGrow: "1",
+    fontFamily: "sans",
+    padding: "8px",
+    selectors: {
+      "&:focus-visible": {
+        outline: "0",
+      },
+    },
+  },
+  variants: {},
+});
+
+export const parentBoxRecipe = recipe({
```

Comment:
let's rename this to
```suggestion
export const wrapper = recipe({
```